packages/terra-clinical-site/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Unreleased
 
 ### Changed
 * Update Header example
+* Added Subheader example
```

Comment:
Can we update the changelog to read something like
`### Added`
`* isSubheader boolean prop to allow rendering of subheader component`